ui/component/settingWalletServer/internal/displayRow.jsx

```diff
@@ -0,0 +1,35 @@
+// @flow
```

Comment:
This file is simple enough to just be included in the `SettingWalletServer` component